dialects/math/math.thorin

```diff
@@ -121,14 +121,15 @@
 ///     * `L`ogarithm
 ///     * `B`ase 2 or natural
 ///
-/// | Subtag | Alias       | B | L | Meaning                 | Semantics      |
-/// |--------|-------------|---|---|-------------------------|----------------|
-/// | `bl`   | `exp`       | o | o | natural exponential     | \f$e^x\f$      |
-/// | `bL`   | `log`       | o | x | natural logarithm       | \f$\ln x\f$    |
-/// | `Bl`   | `exp2`, `b` | x | o | exponential with base 2 | \f$2^x\f$      |
-/// | `BL`   | `log2`      | x | x | logarithm with base 2   | \f$\log_2 x\f$ |
-///
-.ax %math.exp(bl = exp, bL = log, Bl = exp2 = b, BL = log2):
+/// | Subtag | Alias       | B | L | T | Meaning                 | Semantics       |
+/// |--------|-------------|---|---|---|-------------------------|-----------------|
+/// | `bl`   | `exp`       | o | o | o | natural exponential     | \f$e^x\f$       |
+/// | `bL`   | `log`       | o | x | o | natural logarithm       | \f$\ln x\f$     |
+/// | `Bl`   | `exp2`, `b` | x | o | o | exponential with base 2 | \f$2^x\f$       |
+/// | `BL`   | `log2`      | x | x | o | logarithm with base 2   | \f$\log_2 x\f$  |
+/// | `BLT`  | `log10`     | x | o | x | logarithm with base 10   | \f$log_{10} x\f$|
+///
+.ax %math.exp(bl = exp, bL = log, Bl = exp2 = b, BL = log2, BLT=log10):
```

Comment:
The `T` parameter deems a bit out of place/artificial.
Maybe the option for the first letter `NBD` for `natural`, `binary`, and `decimal` would be a better fit.

Reply:
I was contemplating something like 
math.exp.l = exp
math.exp.L = natural log
math.exp.lt = exp2
math.exp.Lt = log2
math.exp.lT = *unused*
math.exp.LT = log10
However, I thought it would be a problem since it wouldn't be compatible with existing code that uses math.exp